pinot-server/src/main/java/org/apache/pinot/server/api/resources/InstanceResource.java

```diff
@@ -73,4 +75,17 @@ public List<String> getInstanceTags() {
     }
     return Collections.emptyList();
   }
+
+  @GET
+  @Path("pools")
+  @ApiOperation(value = "Tenant pools for current instance")
+  @ApiResponses(value = {
+      @ApiResponse(code = 200, message = "Success"), @ApiResponse(code = 500, message = "Internal server error")
+  })
+  @Produces(MediaType.APPLICATION_JSON)
+  public Map<String, String> getInstancePools() {
+    InstanceConfig instanceConfig = HelixHelper.getInstanceConfig(_helixManager, _instanceId);
```

Comment:
need null check for instanceConfig (similar to getInstanceTags)

Reply:
Thanks Ankit, addressed